plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/settings.tsx

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
It should belong to the quantity block [#53009](https://github.com/woocommerce/woocommerce/issues/53009)